src/pudl/transform/ferc1.py

```diff
@@ -2477,6 +2573,54 @@ def spot_fix_rows(self, df: pd.DataFrame) -> pd.DataFrame:
         return df
 
 
+class ElectricEnergyAccountSourcesTransformer(Ferc1AbstractTableTransformer):
```

Comment:
The naming convention we've had for the other classes includes `Ferc1TableTransformer` as the end of the class name (indicating that it descends from the `Ferc1AbstractTableTransformer` rather than a generic `TableTransformer`